lib/sse_test_server/sse_server.ex

```diff
@@ -88,12 +88,17 @@ defmodule SSETestServer.SSEServer do
     update_env(%State{state | sse_endpoints: new_endpoints})
   end
 
-  defp update_env(state) do
-    handlers =
-      state.sse_endpoints
+  defp handlers_for_endpoints(sse_endpoints \\ %{}) do
+    sse_handlers =
+      sse_endpoints
       |> Map.values
       |> Enum.map(&SSEEndpoint.to_handler/1)
       |> Enum.sort
```

Comment:
Err.. why is sorting necessary again?

Reply:
The routes are checked in sequence and the first match is used. Because of this, routes that are suffixes of other routes need to appear first, so we actually need a *reverse* sort. This needs a test.

Reply:
... actually, we don't need *any* sorting. Since there are no optional components in any of the endpoint paths, exactly one will match. The only restriction is that the catchall handler needs to be at the end.